examples/getting_start_with_tensorbay.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+#
+# Copyright 2021 Graviti. Licensed under MIT License.
+#
+
+# pylint: disable=wrong-import-position,wrong-import-order,not-callable,ungrouped-imports
+
+"""This file includes the python code of getting_start_with_tensorbay.rst."""
+
+# =============== Authorize a Client Object ===============
```

Comment:
The this comments act as section separator, this format is not good due to the length of `=` is variable.
Is that possible to use a **fixed** format for section separator?